gmso/tests/test_mcf.py

```diff
@@ -381,11 +416,86 @@ def test_in_cassandra(self, typed_ethane):
             system=system,
             moveset=moveset,
             run_type="equilibration",
-            run_length=100,
+            run_length=0,
+            temperature=300.0 * u.K,
+            run_name="nvt_mbuild",
+            seeds=[12356, 64321],
+        )
+
+        py, fraglib_setup, cassandra = detect_cassandra_binaries()
+
+        # TODO: not sure why the cassandra MCF writer of mBuild
+        # outputs a different intramolecular exclusions relative
+        # to the GMSO writer. This is a temporary fix to make the
+        # test pass. We should investigate this further.
+        # Also, try to use the function top.set_lj_scale() and see how
+        # to update subtopologies.
```

Comment:
Is this something we need to look into now?

Reply:
I do not think this is something we have to worry about for now.